docs/source/performance/apq.md

```diff
@@ -150,16 +150,18 @@ How exactly this works depends on exactly which CDN you chose. Configure your CD
 
 ## Cache configuration
 
-Inside Apollo Server, the automated persisted query registry is stored in a user-configurable cache.  A local in-memory cache is enabled by default, however, if a general default `cache` has been provided as a top-level option to the `ApolloServer` constructor options, that cache will be used instead.
+By default, Apollo Server stores its APQ registry within its local in-memory cache. If you provide a different `cache` as a top-level option to the `ApolloServer` constructor, Apollo Server uses that cache instead.
 
-To override a the cache implementation specifically for automated persisted queries, leaving the general cache untouched, provide an instance of the desired implementation to the `ApolloServer` constructor option for `persistedQueries`.  The available classes and the npm modules they are available from are as follows:
+You can also designate a cache _specifically_ for the APQ registry. To do so, provide an instance of your preferred cache class to the `ApolloServer` constructor as the `persistedQueries` option. The following backing data stores are supported:
 
-* Local, in-memory cache (default, when not specified)
-* Memcached: `MemcachedCache` from [`apollo-server-cache-memcached`](https://npm.im/apollo-server-cache-memcached)
-* Redis (single instance or Sentinels): `RedisCache` from [`apollo-server-cache-redis`](https://npm.im/apollo-server-cache-redis)
-* Redis Cluster: `RedisClusterCache` from [`apollo-server-cache-redis`](https://npm.im/apollo-server-cache-redis)
+| Data store  | Class name  | Library  |
+|---|---|---|
+| Local in-memory cache (default)  | `InMemoryCache`  | [`apollo-cache-inmemory`](https://npm.im/apollo-cache-inmemory)  |
```

Comment:
Ah, the `InMemoryCache` is actually an Apollo Client package.  For the server, this would be:

```suggestion
| Local in-memory cache (default)  | `InMemoryLRUCache`  | [`apollo-server-caching`](https://npm.im/apollo-server-caching)  |
```